Makefile

```diff
@@ -25,9 +25,11 @@ LDFLAGS=-ldflags "-X ${LDFLAG_LOCATION}.version=${VERSION} -X ${LDFLAG_LOCATION}
 GOLANGCI_VERSION := v1.15.0
 HAS_GOLANGCI := $(shell which golangci-lint)
 
+# REGISTRY is the container registry to push into.
+REGISTRY?=staging-k8s.gcr.io
+
 # IMAGE is the image name of descheduler
-# Should this be changed?
-IMAGE:=descheduler:$(VERSION)
+IMAGE:=$(REGISTRY)/descheduler:$(VERSION)
```

Comment:
I'd say, just keep existing IMAGE, and create IMAGE-GCLOUD, and during push, just tag the IMAGE to IMAGE-GCLOUD.

It would help keeping with existing functionality.

Reply:
Fixed